packages/core/src/api/message.ts

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable jsdoc/check-param-names */
+import { makeURLSearchParams, type RawFile, type REST } from '@discordjs/rest';
+import {
+	Routes,
+	type APIMessage,
+	type APIUser,
+	type RESTGetAPIChannelMessageReactionUsersQuery,
+	type RESTPostAPIChannelMessageJSONBody,
+} from 'discord-api-types/v10';
+
+export class MessagesAPI {
+	public constructor(private readonly rest: REST) {}
+
+	/**
+	 * Sends a message in a channel
+	 *
+	 * @param channelId - The id of the channel to send the message in
+	 * @param message - The options to use when sending the message
+	 */
+	public async send(channelId: string, { files, ...body }: RESTPostAPIChannelMessageJSONBody & { files?: RawFile[] }) {
```

Comment:
```suggestion
	public async create(channelId: string, { files, ...body }: RESTPostAPIChannelMessageJSONBody & { files?: RawFile[] }) {
```
Why `message.send()`? We don't send something to a message. We create it.